randovania/layout/game_patches_serializer.py

```diff
@@ -51,7 +51,7 @@ def _pickup_assignment_to_item_locations(
         else:
             item_name = _ETM_NAME
 
-        items_locations[region.name][region_list.node_name(node)] = item_name
+        items_locations[region.name][f"{node.identifier.area}/{node.identifier.node}"] = item_name
```

Comment:
isn't this just `node.full_name()`?

Reply:
Without region, yes. But I wanted to not use a thing mostly for visual feedback into the core serialization function.

Reply:
add a comment then pls